tests/docker/ducktape-deps/kgo-verifier

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 set -e
-git -C /opt clone https://github.com/redpanda-data/kgo-verifier.git
 cd /opt/kgo-verifier
-git reset --hard 319c69da3d7690e26d390261c4b331703b99260c
 go mod tidy
 make
```

Comment:
use bazel here to build the verifier

Reply:
i followed this from rp-storage-tool; we don't pull bazel for building these; should we? 🤔 i thought it would be more heavy weight but didn't validate that guess